provisioning/baremetal_pod_test.go

```diff
@@ -212,7 +212,7 @@ func TestProxyAndCAInjection(t *testing.T) {
 		},
 		{
 			name:       "metal3 containers have proxy and CA information",
-			containers: newMetal3Containers(&images, managedProvisioning().build(), &proxy),
+			containers: newMetal3Containers(&images, managedProvisioning().build(), &proxy, ""),
 		},
 	}
 	for _, tc := range tCases {
```

Comment:
Can we add a new test for when the sshKey is not ""?

Reply:
will upload it now